swarm/storage/chunker_test.go

```diff
@@ -199,7 +199,7 @@ func testRandomData(splitter Splitter, n int, tester *chunkerTester) Key {
 	input, found := tester.inputs[uint64(n)]
 	var data io.Reader
 	if !found {
-		data, input = testDataReaderAndSlice(n)
+		data, input = generateRandomData(n)
```

Comment:
I think this is a better name for this function. We can easily see the return types, but from the current name it is not clear what it actually does.